.github/workflows/test.yml

```diff
@@ -1,12 +1,12 @@
-name: Unit tests
+name: Integration
```

Comment:
🤔

Reply:
For shared first name

<img width="214" alt="image" src="https://user-images.githubusercontent.com/12021443/161990966-ae44f4fd-1902-4272-8930-87624c4599c0.png">

Reply:
fixed